index.src.html

```diff
@@ -670,72 +670,75 @@ <h3 id="settings-object">
     Is |settings object| a secure context?
   </h3>
 
-  Given a <a>settings object</a> (|settings object|), this algorithm returns
+  Given a <a>settings object</a>, |settings object|, this algorithm returns
```

Comment:
You mean instead of using parentheses? Either way, let's change the naming separately.